src/main/java/com/fabriccommunity/spookytime/registry/SpookyFeatures.java

```diff
@@ -0,0 +1,25 @@
+package com.fabriccommunity.spookytime.registry;
+
+import com.fabriccommunity.spookytime.SpookyTime;
+import com.fabriccommunity.spookytime.world.feature.LargeSkeletalTreeFeature;
+import com.fabriccommunity.spookytime.world.feature.SmallSkeletalTreeFeature;
+import net.minecraft.util.registry.Registry;
+import net.minecraft.world.gen.feature.DefaultFeatureConfig;
+import net.minecraft.world.gen.feature.Feature;
+import net.minecraft.world.gen.feature.FeatureConfig;
+
+/**
+ * @author Indigo Amann
+ */
+public class SpookyFeatures {
+    public static Feature<DefaultFeatureConfig>
+            SMALL_SKELETON_TREE,
+            LARGE_SKELETON_TREE;
+    public static void init() {
+        SMALL_SKELETON_TREE = register("skeleton_tree_small", new SmallSkeletalTreeFeature(DefaultFeatureConfig::deserialize, false));
+        LARGE_SKELETON_TREE = register("skeleton_tree_large", new LargeSkeletalTreeFeature(DefaultFeatureConfig::deserialize, false));
+    }
```

Comment:
Can we change this to:
   - 2 separate fields
   - initialized on the same line as the declaration (leave `init` empty)